### PR TITLE
Add information about last modification date

### PR DESCRIPTION
**Pull request by kamilchm**

### CODE REVIEW

**Comment by mattfarina** [comment]
Thanks for the pull request. I'll take a look at it.

**Comment by kamilchm** [comment]
I wasn't sure how to name it :/

**Comment by technosophos** [comment]
I like this feature!

File: bzr.go [hunk 1]
```diff
@@ -94,6 +95,20 @@ func (s *BzrRepo) Version() (string, error) {
 	return strings.TrimSpace(string(out)), nil
 }
 
+// Date retrieves last commit date.
+func (s *BzrRepo) Date() (time.Time, error) {
+	out, err := s.runFromDir("bzr", "version-info", "--custom", "--template={date}")
+	if err != nil {
+		return time.Time{}, err
+	}
+	const longForm = "2006-01-02 15:04:05 -0700"
```
Comment (mattfarina) [inline bzr.go]: It appears the `const longForm` is used in each implementation. Can the definition be moved to the `repo.go` file to avoid duplication.

**Comment by mattfarina** [comment]
Meh, the small changes can be follow on commits. Merged. Thanks!

**Comment by mattfarina** [comment]
@kamilchm I just released 1.3.0 with this in it. Thanks.

File: bzr.go (hunk 1 above)
Comment (kamilchm) [inline bzr.go]: Date format is the same for bzr, git and hg, but different for svn and I don't see any switch in svn that can change it.

**Pull request merged**